doc/internals/todo.md

```diff
@@ -0,0 +1,74 @@
+Todo for Source
+===============
+
+Case expressions and ifs
+------------------------
+
+We probably need slightly clunkier syntax than Haskell, unless we want to implement full-blown whitespace sensitivity.
+I think something like this would be fine.
```

Comment:
fwiw, the Idris parser is [whitespace sensitive](https://github.com/idris-lang/Idris-dev/blob/master/src/Idris/ParseHelpers.hs#L442) and uses kmett's trifecta parser combinators